lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.6.43",
+  "version": "0.6.44",
```

Comment:
I'd prefer to do this 0.7.0 Not sure why we don't use a wider range of version to indicate how big the change is.

Reply:
Good point. Will update to 0.7.0.